### ISSUE TITLE
Short error message workaround should be listed in readme

### ISSUE THREAD

**Issue by br4e** [cgewecke/buidler-gas-reporter#21]
Documentation should note that starting a node is needed when using buidler when the following error is printed on run: 

`  1) Uncaught error outside test suite`

with no additional information.

**Comment by fvictorio** [comment]
@cgewecke if there's no way to make the plugin work with the in-process buidler EVM, maybe something like this could be added to the plugin?

```
      if (options.enabled && network.name !== 'buidlerevm') {
```

It's a hack, but it's probably better than failing without further information.

**Comment by cgewecke** [comment]
This should finally be resolved with `0.1.4` (just released). Very sorry about the delays here.

There's no longer an independent node requirement. The reporter works out of the box with regular BuidlerEVM, requires no special config, is much faster, and all the other BuidlerEVM features should work as well.

Closing here, please just lmk if you run into any problems / bugs when upgrading...
